JAICore/jaicore-basic/src/test/java/jaicore/testproblems/npuzzle/NPuzzleState.java

```diff
@@ -149,20 +145,22 @@ public int getNumberOfWrongTiles() {
 	 */
 	public double getDistance() {
 		double d = 0.0;
-		for(int i = 0; i < board.length; i++) {
-			for(int j = 0;  j < board.length; j++) {
-				int tile = board[i][j];
-				double x = (double)tile / board.length;
-				double y = tile % board.length-1;
-				if(x%1==0)
+		for(int i = 0; i < this.board.length; i++) {
+			for(int j = 0;  j < this.board.length; j++) {
+				int tile = this.board[i][j];
+				double x = (double)tile / this.board.length;
+				double y = tile % this.board.length-1;
+				if(x%1==0) {
 					x--;
+				}
 				x =Math.floor(x);
-				if (y < 0)
-					y = board.length-1;
+				if (y < 0) {
+					y = this.board.length-1;
+				}
 
 				if(tile == 0) {
-//					x = board.length-1;
-//					y = board.length-1;
+					//					x = board.length-1;
```

Comment:
Doesn't seem to be important. So either it's art or it can be omitted.

Reply:
gone